python/librmm_cffi/wrapper.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+from numba import cuda 
+import ctypes
+
+class RMMError(Exception):
```

Comment:
Wouldn't this be better as a subclass of `MemoryError`?

Reply:
Is GDFError a subclass of anything? I copied this from GDFError.